Cargo.toml

```diff
@@ -35,7 +35,11 @@ default = []
 # if enabled, include API for interfacing with chrono 0.4
 chrono-0_4 = ["chrono"]
 # if enabled, include API for interfacing with uuid 0.8
-uuid-0_8 = []
+# This is commented out because Cargo implicitly adds this feature since
+# uuid-0_8 is also an optional dependency.
+# uuid-0_8 = []
+# if enabled, include API for interfacing with uuid 1.x
+uuid-1 = []
```

Comment:
Since uuid is stable and the flags aren't mutually exclusive, we could just include this support out of the box without a feature flag. We may have had a discussion about this already though that I can't remember the conclusions of. Given that we export a `bson::Uuid` type maybe its not necessary.

Reply:
It's not precisely equivalent, but the stumbling blocks we've run into in the driver with things that would have been better as optional features from the start has me inclined to keep this as optional.